spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -7,6 +7,7 @@
         enrollment_established_at: Time.zone.now - 3.days,
         status_check_attempted_at: Time.zone.now - 15.minutes,
         status_updated_at: Time.zone.now - 2.days,
+        issuer: ServiceProvider.find_by(issuer: 'http://localhost:3000').issuer,
```

Comment:
Couple thoughts:

* Rather than updating this at the time of running assertions, could we assign it in the [creation of the stubbed enrollment](https://github.com/18F/identity-idp/blob/128f0d6025ca2ccf89478be332dfa3d405a56851/spec/jobs/get_usps_proofing_results_job_spec.rb#L131)?
* If we know issuer string already, do we need to run a query of the service provider, or could we just use the string `'http://localhost:3000'` directly?